### ISSUE TITLE
Scrollbar on small screen sizes

### ISSUE THREAD

**Issue by moevbiz** [NickPiscitelli/Glider.js#147]
I've just created a slider on my page in no time, this is very pleasant to use! 👍  :~)
 
There's a small scroll bar/indicator which appears on small window sizes, which seems to be connected to my breakpoint settings. It also appears on the demo website on all elements, just below 600px width. 

It doesn't seem to be generated by the browser, but I couldn't find it mentioned in the docs, nor am I able to select it with CSS. Can you tell me how I can apply styles to it, or if there's a way to disable it? Thanks!

**Comment by ScreenTechnicals** [comment]
Yeah that's true that a horizontal scroll bar appears in the mobile layouts !
